test/bdd/agent_controller_steps.go

```diff
@@ -37,21 +38,24 @@ const (
 // AgentWithControllerSteps
 // TODO all steps in this package needs to be placed in different packages instead of sharing same package [Issue #584]
 type AgentWithControllerSteps struct {
-	bddContext     *Context
-	controllerURLs map[string]string
-	webhookURLs    map[string]string
-	invitations    map[string]*didexchange.Invitation
-	connectionIDs  map[string]string
+	bddContext          *Context
+	controllerURLs      map[string]string
+	webhookURLs         map[string]string
+	invitations         map[string]*didexchange.Invitation
+	connectionIDs       map[string]string
```

Comment:
Same for here

Reply:
https://github.com/hyperledger/aries-framework-go/pull/667#discussion_r341698224